src/gg.regression.unity.bots/Runtime/Scripts/StateRecorder/BotSegments/Models/MonkeyBotActionData.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Newtonsoft.Json;
+using RegressionGames.ActionManager;
+using RegressionGames.GenericBots;
+using RegressionGames.StateRecorder.BotSegments.JsonConverters;
+using RegressionGames.StateRecorder.JsonConverters;
+using RegressionGames.StateRecorder.Models;
+using UnityEngine;
+
+namespace RegressionGames.StateRecorder.BotSegments.Models
+{
+    [Serializable]
+    [JsonConverter(typeof(MonkeyBotActionDataJsonConverter))]
+    public class MonkeyBotActionData : IBotActionData
+    {
+        [NonSerialized] 
+        public static readonly BotActionType Type = BotActionType.MonkeyBot;
+        
+        public int apiVersion = SdkApiVersion.VERSION_5;
```

Comment:
This should be a new version 6 as 6 adds type monkey bot support

Reply:
Updated to version 6.